src/vendor/errors.jsx

```diff
@@ -1,154 +1,255 @@
 /* eslint-disable react/no-multi-comp */
+/* eslint-disable max-len */
 import React from 'react';
-import PropTypes from 'prop-types';
-import { missing } from './utils';
+import ErrorPanel from '@mozilla-frontend-infra/components/ErrorPanel';
+import { coalesce, exists, isChrome, isNode, isString, missing } from './utils';
```

Comment:
How are all of these changes related to wrapping components with an HOC? (up to line 200)